keel-core-test/src/main/kotlin/com/netflix/spinnaker/keel/persistence/DeliveryConfigRepositoryTests.kt

```diff
@@ -201,33 +238,7 @@ abstract class DeliveryConfigRepositoryTests<T : DeliveryConfigRepository, R : R
     context("storing a delivery config with artifacts and environments") {
       deriveFixture {
         copy(
-          deliveryConfig = deliveryConfig.copy(
-            artifacts = setOf(
-              artifact, artifactFromBranch
-            ),
-            environments = setOf(
-              Environment(
-                name = "test",
-                resources = setOf(
-                  resource(kind = parseKind("ec2/cluster@v1")),
-                  resource(kind = parseKind("ec2/security-group@v1"))
-                )
-              ),
-              Environment(
-                name = "staging",
-                constraints = setOf(
-                  DependsOnConstraint(
-                    environment = "test"
-                  ),
-                  ManualJudgementConstraint()
-                ),
-                resources = setOf(
-                  resource(kind = parseKind("ec2/cluster@v1")),
-                  resource(kind = parseKind("ec2/security-group@v1"))
-                )
-              )
-            )
-          )
+          deliveryConfig = completeDeliveryConfig
```

Comment:
What's the thinking behind making this a top-level val? It seems to only be used here.

Reply:
I kept this change by accident. Was planning on using that elsewhere and never did. Will undo.